EpikCord/__init__.py

```diff
@@ -2285,7 +2316,16 @@ async def fetch_reply(self):
         return await response.json()
 
     async def edit_reply(self, *, tts: bool = False, content: Optional[str] = None, embeds: Optional[List[Embed]] = None, allowed_mentions = None, flags: Optional[int] = None, components: Optional[List[Union[MessageButton, MessageSelectMenu, MessageTextInput]]] = None, atachments: Optional[List[Attachment]] = None):
+        message_data = {
```

Comment:
Please remove this. This will raise a `400` because that is invalid data to use "`None`"